config/unicorn.rb

```diff
@@ -13,9 +13,19 @@
 preload_app true
 @sidekiq_pid = nil
 
-before_fork do |_server, _worker|
+before_fork do |server, worker|
```

Comment:
Unused block argument - `worker`. If it's necessary, use `_` or `_worker` as an argument name to indicate that it won't be used.